src/docker/compose.dss-global.yaml

```diff
@@ -13,3 +13,4 @@ services:
       extends:
         file: compose.processor-template.yaml
         service: processor-template
+      restart: unless-stopped
```

Comment:
Will you please move this to `compose.processor-template.yaml`?

That way it will be used for both a global DSS (on a public chain) and for a local DSS (on a local testnet)

Reply:
Thank you for your feedback, I moved it to `compose.processor-template.yaml`